src/site/elements/pages/rester-page-request.js

```diff
@@ -935,6 +940,11 @@ class RESTerPageRequest extends RESTerLintMixin(
         dialogs.curlCommand.show(compiledRequest);
     }
 
+    _openRequestInNewTab() {
+        const { compiledRequest } = this._compileRequest();
```

Comment:
Can you also add `this.$.moreOptions.close();` here to close the menu? That would make it consistent with the other menu items.